source/deployment/guest-accounts.rst

```diff
@@ -0,0 +1,125 @@
+.. _guest-accounts:
+
+Guest Accounts (Beta) (E10 and E20)
+===================================
+
+Available in `Enterprise Edition E10 and higher <https://about.mattermost.com/pricing/>`__. 
+
+Guest Accounts are a new role within Mattermost Enterprise editions which allow organizations to collaborate with individuals (such as vendors and contractors) outside of their organization. The Guest Account feature controls access to channels and other members within the organization. 
+
+An organization may use guest accounts to work on a website project with resources from external design firm.  Or Guest accounts could be used to collaborate with customers on a support issue.  
+
+Guests can: 
```

Comment:
This and the below don't appear as lists. Probably need to add a space after the "heading" and the numbered list.

Reply:
meant for these to be bullets.  Updated